src/styles/card.css

```diff
@@ -0,0 +1,82 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 16px;
+  justify-content: space-evenly;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  color: var(--main-color);
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__name {
+  display: inline-block;
```

Comment:
why would you need to make it an inline-block? it does not seem that you set the width or height
write less code